src/content/protocols/aave.md

```diff
@@ -0,0 +1,998 @@
+| Emergency Admin                     | [0x2cfe3ec4d5a6811f4b8067f0de7e47dfa938aa30](https://etherscan.io/address/0x2cfe3ec4d5a6811f4b8067f0de7e47dfa938aa30) | Multisig 5/9 | ✅                 | ✅                     | ❌                       | ✅ ([source](https://aave.com/docs/primitives/governance#community-guardians-governance-emergency-guardian))                                |
+| Risk Council (for Pool Stewards)    | [0x47c71dFEB55Ebaa431Ae3fbF99Ea50e0D3d30fA8](https://etherscan.io/address/0x47c71dFEB55Ebaa431Ae3fbF99Ea50e0D3d30fA8) | Multisig 2/2 | ❌                 | ✅                     | ❌                       | ✅ ([source](https://vote.onaave.com/proposal/?proposalId=197&ipfsHash=0x2f41406557b0fc69c256916c066a77c57434fa77ccab3cfe56c8db6a4f306c01)) |
+| Risk Council (for GhoGsmSteward)    | [0x8513e6F37dBc52De87b166980Fa3F50639694B60](https://etherscan.io/address/0x8513e6F37dBc52De87b166980Fa3F50639694B60) | Multisig 3/4 | ❌                 | ✅                     | ❌                       | ✅ ([source](https://vote.onaave.com/proposal/?proposalId=61&ipfsHash=0x111fca5019e0bc411dc4bfa36167ad7d88c57c9ddbffb202fb1568e16dd06570))  |
+| CrossChainController Guardian (BGD) | [0xb812d0944f8F581DfAA3a93Dda0d22EcEf51A9CF](https://etherscan.io/address/0xb812d0944f8F581DfAA3a93Dda0d22EcEf51A9CF) | Multisig 2/3 | ❌                 | ✅                     | ❌                       | ✅ ([source](https://github.com/bgd-labs/aave-permissions-book/blob/main/out/MAINNET-V3.md#governance-v3-contracts))                        |
+| CleanUp Admin                       | [0xdeadD8aB03075b7FBA81864202a2f59EE25B312b](https://etherscan.io/address/0xdeadD8aB03075b7FBA81864202a2f59EE25B312b) | Multisig 2/3 | ❌                 | ✅                     | ❌                       | ✅ ([source](https://vote.onaave.com/proposal/?proposalId=270&ipfsHash=0x4043001b72316afa6b6728772941bfa08f127b66c1c006316a3f20510b6738ab)) |
+| Aave Chan Initiative (ACI)          | [0xac140648435d03f784879cd789130F22Ef588Fcd](https://etherscan.io/address/0xac140648435d03f784879cd789130F22Ef588Fcd) | Multisig 1/2 | ❌                 | ❌                     | ❌                       | ✅                                                                                                                                          |
+| ACI Automation (Bot)                | [0x3Cbded22F878aFC8d39dCD744d3Fe62086B76193](https://etherscan.io/address/0x3Cbded22F878aFC8d39dCD744d3Fe62086B76193) | EOA          | ❌                 | ❌                     | ❌                       | ❌                                                                                                                                          |
+
+💡 The BGD Labs maintains a public markdown page on the existing permissions to inform its users: https://github.com/bgd-labs/aave-permissions-book/blob/main/out/ARBITRUM_ONE-V3.md
+
+## Councils and their permissions
```

Comment:
Make section "Multisigs and their permissions" and clean up wording around these: we only speak of "Security Council" for multisigs adhering to the requirements. otherwise we only say Multisig or Multisig account

Reply:
alright 👍 I did said "Council" bc they speak of "Risk Council"